db/seeds.rb

```diff
@@ -1,42 +1,42 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Jobtag.destroy_all
+ActiveRecord::Base.connection.reset_pk_sequence!("jobtags")
```

Comment:
Agreed, we would have to make sure everyone runs `rake db:reset` rather than `setup`, but I am fine with that